### DISCUSSION TITLE
Would a "Find a preaching partner" feature be doable?

### DISCUSSION THREAD

**Discussion by ux-git** [sws2apps/organized-app#793] [Ideas]
I'm thinking about a feature where:
1) Publisher can send a request (and set desired time frame or set a flexible)
2) Other publishers will see the requests in the shared congregation list 
3) Anyone can click "Work together" (or something like that) and then personally discuss the details.

Does this sound like something possible to implement?
<img width="974" alt="Screenshot 2023-12-19 at 14 11 27" src="https://github.com/sws2apps/cpe-sws/assets/80993061/f6108bd3-ff0f-4b98-8c8c-9a7f2aa96b79">

**Comment by rhahao** [comment]
Hello @ux-git. Yes I believe it should be doable. So we can have this in the design and think of its implementation during development. Thanks.
